packages/cloudforet/language-pack/en.json

```
@@ -972,7 +972,7 @@
 				"DAY": "Day",
 				"MONTH": "Month",
 				"NETWORKING": "Networking",
-				"NO_SERVICE": "No {service}",
+				"NO_SERVICE": "No Database",
```

```suggestion
				"NO_SERVICE": "No {service}",
```

This is [custom formatting](https://kazupon.github.io/vue-i18n/guide/formatting.html#named-formatting) of i18n :)
It's used in this way.
```typescript
$t('COMMON.WIDGETS.ALL_SUMMARY.NO_SERVICE', { service: labelMap[activeTab] })
```